src/frontend/nginx.conf

```diff
@@ -1,32 +1,25 @@
-events {
-  worker_connections  256;
-}
+worker_processes 4;
+
+events { worker_connections 1024; }
 
 http {
     include   /etc/nginx/mime.types; # fix it as per your env
+       
     server {
-        root dist/; # necessary to avoid nginx injecting root into try_files directives
-
-        listen 8080;
-        server_name localhost;
-        server_name 127.0.0.1;
-        sendfile off;
-        
-        location / {
-            try_files $uri$args $uri$args/ /index.html;
-        }
+      listen       80;
+      server_name  localhost;
 
-        # serve static content
-        location /static/ {
-            # path for static files
-            root .;
-        }
+      location / {
+          root   /usr/share/nginx/html;
```

Comment:
is this part of server or location?

Reply:
and will this allow to run withOUT docker? (as we used to run of any OS with this conf file on mac?